dataplicity/m2mmanager.py

```diff
@@ -153,8 +155,17 @@ def on_instruction(self, sender, data):
             m2m_port = data['m2m_port']
             self.client.port_forward.redirect_port(device_port, m2m_port)
         elif action == 'reboot-device':
-            log.debug('reboot requested')
             self.reboot()
+        elif action == 'read-file':
```

Comment:
The number of actions is getting a little unwieldily, and more are planned. Will consider some kind of dispatch mechanism in a future update.